src/structure/pdb.jl

```diff
@@ -107,6 +321,17 @@ function Base.read(filepath::AbstractString,
     end
 end
 
+# Read PDB file based on PDB ID and pdb_dir. 
+function Base.read(pdbid::AbstractString,
```

Comment:
Actually I just added it because that will be in a uniform format as the rest. If users wants to download first and then read it later, they will have uniform way of calling the functions. We may change the arguments as you mentioned.